tests/testsuite.src/syn_copy.at

```diff
@@ -954,6 +946,51 @@ AT_DATA([prog.cob], [
        REPLACE OFF.
 ])
 
-AT_CHECK([$COMPILE_ONLY prog.cob], [0], [], [])
+AT_CHECK([$COMPILE_ONLY -P=preproc.lst prog.cob])
```

Comment:
Not sure - can we use` `-P-` as with the listings? If not it likely won't be much work to copy that over...

Reply:
I added a commit for that.